sdk/monitor/azure-monitor-query/azure/monitor/query/_logs_query_client.py

```diff
@@ -35,6 +30,11 @@ class LogsQueryClient(object):
         :language: python
         :dedent: 0
         :caption: Creating the LogsQueryClient with a TokenCredential.
+
+    :param credential: The credential to authenticate the client
```

Comment:
nit, consistency with endpoint formatting
```suggestion
    :param credential: The credential to authenticate the client.
```